content/scout/policy/_index.md

```diff
@@ -214,3 +215,19 @@ in the CLI.
 > is enabled. Docker Scout doesn't have access to historic evaluations. Trigger
 > a SonarQube analysis and policy evaluation after enabling the integration to
 > view the results in Docker Scout.
+
+### Default user is root
```

Comment:
I'd like to have a call out somewhere in this doc that says that this policy does not guarantee that the container will not run as a non-root user. It evaluates whether the image config blob has a root user specified (i.e. by setting `USER root` or omitting it in the `Dockerfile`).

**People can still override the user at runtime** when doing `docker run --user <user>` or in Kubernetes in the Pod spec with `runAsUser`.

Reply:
added a note about this, PTAL 🙏🏻